src/routers/components/addingBook.jsx

```diff
@@ -33,7 +33,7 @@ ReloadList.propTypes = {
   bookData: PropTypes.shape({
     title: PropTypes.string.isRequired,
     author: PropTypes.string.isRequired,
-    item_id: PropTypes.number.isRequired,
+    item_id: PropTypes.string.isRequired,
   }).isRequired,
 };
 
```

Comment:
![image](https://github.com/Zven94/Bookstore/assets/83042742/2e4ec822-7f2c-4914-a4ad-715666962855)
- [ **OPTIONAL** ] `src/routers/components/newBook.jsx` For good UX experiences and for best practices, I think it would be better if you add the `required` attribute in each `input` tag to prevent users submitting empty forms. It can lok to something like this:
![Screenshot from 2023-07-21 06-46-35](https://github.com/Zven94/Bookstore/assets/83042742/f8240b51-db8f-4b54-931c-4f3be7b47cf4)